mindsdb/repository.yaml

```diff
@@ -0,0 +1,6 @@
+name: mindsdb
+description: mindsdb deployed on plural
+category: DATA
+private: true
+icon: plural/icons/mindsdb.png
+notes: plural/notes.tpl
```

Comment:
you should add yourself as a contributor here

Reply:
okay